posts/zzz_DO_NOT_EDIT_admiral_1.3.../admiral_1.3.0:_documentation_go!!.qmd

```diff
@@ -0,0 +1,163 @@
+# Note that the date below will be auto-updated when the post is merged.
+date: "2025-06-20"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [ADaM]
+# Feel free to change the image
+image: "pharmaverse.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_admiral_1.3..."
+library(link)
+link::auto()
+```
+
+<!--------------- post begins here ----------------->
+
+The second half of 2025 brings us once again to an exciting {admiral} release - doesn't time fly?! Read through this blog post to learn more what to expect in {admiral} 1.3.
+
+## Headline enhancement
+
+Our crowing update this release cycle is...
+
+**Documentation updates!**
+
+Hmmm, doesn't sound fancy enough? What about...
+
+<span style="font-size: 2em;
+              background: linear-gradient(to right, red, orange, yellow, green, blue, indigo, violet);
+              -webkit-background-clip: text;
+              -webkit-text-fill-color: transparent;">
+Documentation updates!!
+</span>
+
+Still not that interesting? Well, let us change your mind with an example. Here's what the examples section of the reference page for admiral::derive_extreme_records() looked like in {admiral} 1.2:
+
+<img src="derive_extreme_records_before.png" align="center" style="float:right" width="100%"/>
+
+And here's what it will look like in {admiral} 1.3:
+
+<img src="derive_extreme_records_before.png" align="center" style="float:right" width="100%"/>
+
+That's right! Through the implementation of a custom roclet, some of our reference examples have now been enhanced such that:
+
+- Each example has a title, which is also shown in the table of contents, and a description explaining what is happening.
+- The examples are often building up in complexity, starting from a simple application of the function in a toy case and building up to more structure use-cases.
+- The examples now cover __all__ the arguments of the function.
+
+As usual, you can access these reference examples from our [documentation website](https://pharmaverse.github.io/admiral/dev/reference/) or alternatively directly from RStudio's own Help pane by typing `?your_function_name` in the console.
+
+For this first iteration, we have targeted some of the most complex, flexible and important functions within {admiral}. As such, at the time of writing, the functions which have received this enhanced examples treatment are.
+
+- admiral::derive_extreme_records()
+- admiral::derive_param_tte()
+- admiral::derive_summary_records()
+- admiral::derive_extreme_event()
+- admiral::derive_vars_joined()
+- admiral::derive_vars_joined_summary()
+- admiral::derive_vars_merged()
+- admiral::derive_vars_crit_flag()
+- admiral::derive_var_trtemfl()
+- admiral::derive_param_computed()
+- admiral::derive_vars_dt()
+- admiral::derive_vars_dt()
+- admiral::derive_var_extreme_flag()
```

Comment:
can we use link package to link to these specifically?

Reply:
or is this how it is done?  i can remember!?!

Reply:
this is how the link pkg expects them